source/simulator/assembly.cc

```diff
@@ -1551,6 +1636,906 @@ namespace aspect
     computing_timer.exit_section();
   }
 
+  template <int dim>
+  void Simulator<dim>::
+  local_assemble_advection_face_terms(const AdvectionField     &advection_field,
+                                      const typename DoFHandler<dim>::active_cell_iterator &cell,
+                                      internal::Assembly::Scratch::AdvectionSystem<dim> &scratch,
+                                      internal::Assembly::CopyData::AdvectionSystem<dim> &data)
+  {
+    const bool use_bdf2_scheme = (timestep_number > 1);
+
+    const unsigned int n_q_points    = scratch.face_finite_element_values.n_quadrature_points;
+
+    // also have the number of dofs that correspond just to the element for
+    // the system we are currently trying to assemble
+    const unsigned int advection_dofs_per_cell = data.local_dof_indices.size();
+
+    Assert (advection_field.is_discontinuous(introspection), ExcMessage("Face terms only need assembling in the case of discontinuous temperature or compositional discretization."));
```

Comment:
break the statement onto 2 lines (e.g., at `ExcMessage`) to roughly fit it into 80 character lines